dbt_platform_helper/COMMANDS.md

```diff
@@ -987,23 +991,20 @@ platform-helper database (dump|load)
 ## Usage
 
 ```
-platform-helper database dump --account-id <account_id> --app <application> 
-                              --env <environment> --database <database> 
-                              --vpc-name <vpc_name> 
+platform-helper database dump --from <from_env> --database <database> 
+                              [--app <application>] [--from-vpc <from_vpc>] 
 ```
 
 ## Options
 
-- `--account-id <text>`
-
 - `--app <text>`
-
-- `--env <text>`
-
+  - The application name. Required unless you are running the command from your deploy repo
+- `--from <text>`
+  - This is required unless you are running the command from your deploy repo
```

Comment:
The environment you are dumping data from. Required unless you are running the command from your deploy repo